tests/TestCase/Controller/CookieComponentControllerTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP Project
+ * @since         3.1.6
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Controller;
+
+use Cake\Controller\Component;
+use Cake\Core\Configure;
+use Cake\Routing\DispatcherFactory;
+use Cake\Routing\Router;
+use Cake\TestSuite\IntegrationTestCase;
+use Cake\Utility\Security;
+
+/**
+ * CookieComponentControllerTest class
+ */
+class CookieComponentControllerTest extends IntegrationTestCase
```

Comment:
This test case should probably be a sibling of `tests/TestCase/TestSuite/`. It looks like an integration test for the integration test case :smile: